src/main/webapp/resources/js/pages/projects/samples-metadata-import/components/SampleMetadataImportMapHeaders.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { useHistory, useParams } from "react-router-dom";
+import { Button, Radio, Typography } from "antd";
+import { SampleMetadataImportWizard } from "./SampleMetadataImportWizard";
+import { BlockRadioInput } from "../../../../components/ant.design/forms/BlockRadioInput";
+import { useSetColumnProjectSampleMetadataMutation } from "../../../../apis/metadata/metadata-import";
+import {
+  IconArrowLeft,
+  IconArrowRight,
+} from "../../../../components/icons/Icons";
+
+const { Text } = Typography;
+
+/**
+ * React component that displays Step #2 of the Sample Metadata Uploader.
+ * This page is where the user selects the sample name column.
+ * @returns {*}
+ * @constructor
+ */
+export function SampleMetadataImportMapHeaders() {
+  const { projectId } = useParams();
+  const history = useHistory();
+  const [column, setColumn] = React.useState();
+  const { headers, sampleNameColumn } = useSelector((state) => state.reducer);
+  const [updateColumn] = useSetColumnProjectSampleMetadataMutation();
+
+  React.useEffect(() => {
+    setColumn(sampleNameColumn ? sampleNameColumn : headers[0]);
+  }, []);
+
+  const onSubmit = () => {
+    updateColumn({ projectId, sampleNameColumn: column })
+      .unwrap()
+      .then((payload) => {
+        history.push("review");
+      });
+  };
+
+  return (
+    <SampleMetadataImportWizard currentStep={1}>
+      <Text>{i18n("SampleMetadataImportMapHeaders.description")}</Text>
+      <Radio.Group
+        style={{ width: `100%` }}
+        value={column}
+        onChange={(e) => setColumn(e.target.value)}
+      >
+        {headers.map((header, index) => (
+          <BlockRadioInput key={`metadata-uploader-radio-header-${index}`}>
+            <Radio
+              key={`metadata-uploader-radio-header-${index}`}
+              value={header}
+            >
+              {header}
+            </Radio>
+          </BlockRadioInput>
+        ))}
+      </Radio.Group>
+      <div style={{ display: "flex" }}>
+        <Button
+          className="t-metadata-uploader-file-button"
+          icon={<IconArrowLeft />}
+          onClick={() => history.goBack()}
+        >
+          {" "}
```

Comment:
Get rid of this empty space.

Reply:
Please see 34cb910e3d0ca2b2b205e0854c2b600b3cd98a7a.